src/go/rpk/go.mod

```diff
@@ -3,88 +3,88 @@ module github.com/redpanda-data/redpanda/src/go/rpk
 go 1.19
 
 require (
-	cloud.google.com/go/compute v1.7.0
-	github.com/AlecAivazis/survey/v2 v2.3.5
-	github.com/avast/retry-go v3.0.0+incompatible
-	github.com/aws/aws-sdk-go v1.44.58
-	github.com/beevik/ntp v0.3.0
-	github.com/cespare/xxhash v1.1.0
-	github.com/coreos/go-systemd/v22 v22.3.2
-	github.com/docker/docker v20.10.17+incompatible
-	github.com/docker/go-connections v0.4.0
-	github.com/docker/go-units v0.4.0
-	github.com/fatih/color v1.13.0
-	github.com/google/uuid v1.3.0
-	github.com/hashicorp/go-multierror v1.1.1
-	github.com/lestrrat-go/jwx v1.2.25
-	github.com/lorenzosaino/go-sysctl v0.3.1
-	github.com/olekukonko/tablewriter v0.0.5
-	github.com/opencontainers/image-spec v1.0.3-0.20211202183452-c5a74bcca799
-	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.37.0
-	github.com/safchain/ethtool v0.2.0
-	github.com/sethgrid/pester v1.2.0
-	github.com/sirupsen/logrus v1.9.0
-	github.com/spf13/afero v1.9.2
-	github.com/spf13/cobra v1.5.0
-	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.7.1
-	github.com/tklauser/go-sysconf v0.3.10
-	github.com/twmb/franz-go v1.9.2-0.20221101035527-a07b8c8a01ce
-	github.com/twmb/franz-go/pkg/kadm v1.3.0
-	github.com/twmb/franz-go/pkg/kmsg v1.2.0
-	github.com/twmb/tlscfg v1.2.0
-	github.com/twmb/types v1.1.6
-	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f
-	golang.org/x/sys v0.0.0-20220715151400-c0bba94af5f8
-	golang.org/x/term v0.0.0-20220526004731-065cf7ba2467
-	gopkg.in/yaml.v3 v3.0.1
+  cloud.google.com/go/compute v1.7.0
```

Comment:
I believe this was vscode locally. I configure it to insert spaces instead of tabs characters, and have my tabwith set to 2 spaces, but when I realized this whole repo is full of hard tabs I reconfigured to display it to me as 2 spaces but not change the underlying file to prevent the PR touching every line. I must have missed reverting the changes made on this file. My apologizes, I'll put reset this file to the upstream version and make sure only the needed changes are in there, if any. I don't recall manually adding any new modules to the file.